### PR TITLE
Create OSDay 25 event

### PR DESCRIPTION
**Pull request by Readpato**
### 🔗 Linked issue

https://github.com/Schroedinger-Hat/schroedinger-hat/issues/273

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Resolves https://github.com/Schroedinger-Hat/schroedinger-hat/issues/273.

To be merged after the publication of the tickets for OSDay 25.

Banner image for event still missing.

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Review by TheJoin95 — APPROVED** [review]
Aapprovef but wait until the Eventbrite publish date to merge it

**Review by Wabri — COMMENTED** [review]
Should also the home page button be change? 

https://github.com/Schroedinger-Hat/schroedinger-hat-website/blob/bf3ca65df822d736908ea047db5865cf0bf4c89b/src/pages/PageHome.vue#L8

I think it is better rather than creating a new pr for it!

**Comment by Readpato** [comment]
Good catch, fixing it

**Review by Wabri — APPROVED** [review]

**Review by TheJoin95 — APPROVED** [review]

**Pull request merged**
